### DISCUSSION TITLE
No way to disable Picocss completely?

### DISCUSSION THREAD

**Discussion by cakir-enes** [AnswerDotAI/fasthtml#593] [Q&A]
`pico.min.css` gets included by default is there a way to remove it, it clashes with the styles I'm trying to use via tailwindcss not even `!important` can help it.

**Comment by jamilraichouni** [comment]
This is how I use TailwindCSS:

```python
app, rt = fast_app(
    hdrs=(
        ft.Script(
            src="static/js/tailwind-3.4.15.js",
        ),
    ),
    pico=False,
    live=True,
)
```
